parent/pom.xml

```diff
@@ -241,11 +241,12 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
```

Comment:
I think we might as well just delete the <includes> completely since the defaults as described at https://eclipse.org/tycho/sitedocs/tycho-surefire/tycho-surefire-plugin/test-mojo.html#includes is "**/Test*.java **/*Test.java **/*TestCase.java"

likewise with excludes (or at least add "**/*$*' to not pick up inner classes)